### PR TITLE
APIGOV-26388 - Stage

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Adding the stage concepts

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Review by lbadenhop — APPROVED** [review]

**Comment by lbadenhop** [comment]
@cbordier, please review edits. Also, is this PR ready to be merged? Is Deleting a stage information complete?

**Comment by cbordier** [comment]
@lbadenhop I like your edits.
No the PR is not ready to merge. There are some work in progress we need to include.

File: content/en/docs/connect_manage_environ/stage.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+A stage is a concept that provides an abstraction view of the lifecycle state of an API (for example, beta, dev, pre-prod, prod, etc.).
+The stage identifies where the API is available and if any restriction apply. For instance, in a dev stage, the API security might be less important than in production stage.
+Once the API satisfies all the criteria of the current stage, the API can be promoted to the next stage, helping you to build your API lifecycle and to ensure a correct implementation at each stage.
+
+{{< alert title="Note" color="primary" >}}Stage management is optional.{{< /alert >}}
+
+## Stage and environment
+
+An environment might be link to a specific gateway managing one or more stages of the API lifecycle.
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/stage.md:22]: typo "might be linked to a specific gateway"

File: content/en/docs/connect_manage_environ/stage.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+Central Admin, Catalog Manager and Developer can view the Stages.
+
+1. Navigate to *Topology > Stages*.
+
+The list of stages is displayed with the following information:
+
+* Stage name - the title of the stage.
+* Environment - the number of environments the stage is associated to.
+* Services - the number of services the stage is associated to.
+* Assets - the number of asset that have a service the stage is associated to.
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/stage.md:61]: typo "number of assets"

**Review by robertengaxway — APPROVED** [review]
I caught a few typos as single line comments but I approve with those typos fixed

**Pull request merged**